src/aggregator/client/options.go

```diff
@@ -175,7 +246,6 @@ func NewOptions() Options {
 		shardFn:                    sharding.DefaultHash.MustShardFn(),
 		shardCutoverWarmupDuration: defaultShardCutoverWarmupDuration,
 		shardCutoffLingerDuration:  defaultShardCutoffLingerDuration,
-		watcherOpts:                placement.NewStagedPlacementWatcherOptions(),
```

Comment:
Just wondering why it's necessary to set watcher opts explicitly rather than have the default here

Reply:
So this is because watcher opts are only used for rawtcp backends, so it's not always required and we should avoid constructing it if it's not needed (for m3msg backend).